src/core/data-tests.js

```diff
@@ -23,11 +23,50 @@ export const name = "core/data-tests";
 const lang = defaultLang in l10n ? defaultLang : "en";
 
 function toListItem(href) {
-  return hyperHTML.bind(document.createElement("li"))`
+  const emojiList = document.createElement("span");
+  const [testFile] = new URL(href).pathname.split("/").reverse();
+  const testParts = testFile.split(".");
+  let [testFileName] = testParts;
+
+  const isSecureTest = testParts.find(part => part === "https");
+  if (isSecureTest) {
+    const requiresConnectionEmoji = hyperHTML.bind(
+      document.createElement("span")
+    )` 🔒 `;
+    requiresConnectionEmoji.setAttribute(
+      "aria-label",
+      "requires a secure connection"
+    );
+    requiresConnectionEmoji.setAttribute("title", "Test requires HTTPS");
+    testFileName = testFileName.replace(".https", "");
+    emojiList.append(requiresConnectionEmoji);
+  }
+
+  const isManualTest = testFileName
+    .split(".")
+    .join("-")
+    .split("-")
+    .find(part => part === "manual");
+  if (isManualTest) {
+    const manualPerformEmoji = hyperHTML.bind(
+      document.createElement("span")
+    )` 💪 `;
```

Comment:
nit: we probably don't need the whitespace. We should add that between the spans. If we then want them to have more padding, we should add that using CSS.

Reply:
ok I'll keep all of that in mind :+1: